tests/scripts/test_config.yaml

```diff
@@ -15,38 +15,44 @@
 # test outputs - at that point you can populate the "tests/outputs" directory, and your next run will result
 # in a 0 return value.
 
-reader_name: @readername@
-sectored_read: @False       # Almost always False, except for ABI or AHI
+reader_name: amsr2_netcdf # @readername@ - appropriate readername
+sectored_read: False       # @ Almost always False, except for ABI or AHI
 
 available_sectors:
   tcweb_bdeck:
     trackfile_parser: bdeck_parser
     trackfiles:
-      - !ENV ${GEOIPS_PACKAGES_DIR}/geoips/tests/sectors/tc_bdecks/@bio012020.dat@      # Identify appropriate TC case
-    tc_template_yaml: !ENV ${GEOIPS_PACKAGES_DIR}/geoips/geoips/yaml_configs/sectors_dynamic/tc_web_template.yaml
+      # @ Identify appropriate TC case
+      # @ Update "trackfiles" to point to appropriate bdeck file with coverage.
+      - !ENV ${GEOIPS_PACKAGES_DIR}/geoips/tests/sectors/tc_bdecks/bio012020.dat
+    # @ This is the default TC Web template - can use a different template if desired.
+    # @ Found in geoips/plugins/yaml/sectors/dynamic
+    sector_spec_generator: tc_web
 
   global:
-    sectorfiles:
-      - !ENV ${GEOIPS_PACKAGES_DIR}/geoips/tests/sectors/static/global.yaml
     sector_list:
       - global
 
 outputs:
   tc_clean:
     requested_sector_type: tcweb_bdeck
-    output_format: imagery_clean
-    filename_formats: ['tc_clean_fname']
+    output_formatter: imagery_clean
+    filename_formatters: ['tc_clean_fname']
     product_names:
-      - @product1@
-      - @product2@
-    compare_path: !ENV ${GEOIPS_PACKAGES_DIR}/@package@/tests/outputs/@datatype@.<output>.<product>
+      - 89-Test  # @product1@ 
+      - 89pct # @product2@
+    # @ Update this path to the directory you want to store your "correct" outputs in.
+    # @ <output> and <product> are auto-populated at runtime, so can remain unchanged.
+    compare_path: !ENV ${GEOIPS_PACKAGES_DIR}/template_basic_plugin/tests/outputs/amsr2.<output>.<product>
 
   global_clean:
     requested_sector_type: global
     minimum_coverage: 1
-    output_format: imagery_clean
-    filename_formats: ['geoips_fname']
-    compare_path: !ENV ${GEOIPS_PACKAGES_DIR}/@package@/tests/outputs/@datatype@.<output>.<product>
+    output_formatter: imagery_clean
+    filename_formatters: ['geoips_fname']
+    # @ Update this path to the directory you want to store your "correct" outputs in.
+    # @ <output> and <product> are auto-populated at runtime, so can remain unchanged.
+    compare_path: !ENV ${GEOIPS_PACKAGES_DIR}/template_basic_plugin/tests/outputs/amsr2.<output>.<product>
     product_names:
-      - @product1@
-      - @product2@
+      - 89-Test # @product2@
```

Comment:
This produces both of the same images as the single_source test (global and TC 89-Test), plus 2 additional images just for fun.

Reply:
Looks good!